Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -815,3 +815,114 @@ use the install commands,
 .. code-block:: bash
 
    spack install amrex@develop dimensions=2 +cuda cuda_arch=60
+
+.. _sec:build:amrex-kokkos:
+
+AMReX--Kokkos
+=============
+There is an `example in amrex-devtests`_ of how one can build `Kokkos`_ and `AMReX`_. This uses ``cmake`` to first compile
+Kokkos and AMReX, and then the example is built. This section describes the build procedure on `Perlmutter`_, though this can be used as
+a template for other machines as well.
+
+Load the following modules, and specify ``MPI_INCLUDE_PATH`` in ``~/.bash_profile``. The ``cmake`` version has to be ``3.24.3``.
+
+.. code-block:: bash
+
+   module load cray-mpich
+   module load PrgEnv-gnu
+   module load cudatoolkit/12.2
+   module load cmake/3.24.3
+
+   export MPI_INCLUDE_PATH=/opt/cray/pe/mpich/8.1.28/ofi/gnu/12.3/include
```

Comment:
A more portable way might be `export CUDAHOSTCXX=CC` or use `-DCMAKE_CUDA_HOST_COMPILER`.